hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.js

```diff
@@ -142,6 +167,19 @@ describe("Connected tables", () => {
         );
     });
 
+    it("Calls reportIncident when the 'Report broken/lost' button is clicked", () => {
```

Comment:
Move this up into the block with the unconnected table, and switch to using the unconnected component. You could add this right inline with the `it("Calls 'push' when 'Report broken/lost' is clicked ")` test, just by adding `expect(reportIncident).toHaveBeenCalled()`.

Once this is a real redux action, it won't be a prop on the connected component, so this test won't end up here anyway.